File: cms-backend/Shared/Models/Content.cs
namespace CMS.Shared.Models;

public class Content
{
    public Guid Id { get; set; } = Guid.Empty;

    public string Image { get; init; } = string.Empty;
    public string Text { get; init; } = string.Empty;

    public Language Language { get; init; } = Language.En;

    public TextAlignment TextAlignment { get; init; } = TextAlignment.Left;
}

File: cms-frontend/src/models/content.ts
export type Alignment = "left" | "center" | "right";
export type LangCode = "en" | "da" | "sv";

export interface EditorValues {
  language: LangCode;
  image: string;
  text: string;
  textAlignment: Alignment;
}

export interface ContentDto {
  id?: string;
  language: LangCode;
  image: string;
  text: string;
  textAlignment: Alignment;
}
